### ISSUE TITLE
Small bug on Schema json method

### ISSUE THREAD

**Issue by malonsocasas** [delta-io/delta-rs#121]
In Schema class, the `json` method is returning wrong attribute:
```
--- a/python/deltalake/schema.py
+++ b/python/deltalake/schema.py
class Schema:
         return self.__str__()
 
     def json(self) -> Dict[str, Any]:
-        return self.json_schema
+        return self.json_value
```

Maybe add a test:
```
def test_schema_json():
    table_path = "../rust/tests/data/simple_table"
    dt = DeltaTable(table_path)
    schema = dt.schema()
    assert schema.json() == {
        "fields": [{"metadata": {}, "name": "id", "nullable": True, "type": "long"}],
        "type": "struct",
    }
```

**Comment by fvaleye** [comment]
Good catch! 👍 

This PR fix the problem: https://github.com/delta-io/delta-rs/pull/122

**Comment by fvaleye** [comment]
Since PR #122 is merged, I close the issue

**Comment by houqp** [comment]
@fvaleye do you want to try doing a python binding release? we just need to bump the version in python cargo.toml, then create a new github release with tag `python-vx.x.x` to trigger the release pipeline once the commit is merged into main.
